packages/filter-workspace-packages/test/parsePackageSelector.ts

```diff
@@ -177,23 +176,21 @@ const fixtures: Array<[string, PackageSelector]> = [
   ],
 ]
 
-test('parsePackageSelector()', (t) => {
+test('parsePackageSelector()', () => {
   for (const fixture of fixtures) {
-    t.deepEqual(
-      parsePackageSelector(fixture[0], process.cwd()),
-      fixture[1],
-      `parsing ${fixture[0]}`
-    )
+    expect(
+      parsePackageSelector(fixture[0], process.cwd())).toStrictEqual(fixture[1])
+    // @zkochan there is no expect message here, this return warnings
+    // console.log(`parsing ${fixture[0]}`);
```

Comment:
not sure I understand what is the issue. Seems like you converted it correctly, just remove the comments

Reply:
I meant `tap` expects allows 3 arguments, the last one a message, while the `expect` jest only allows 2. This is the unique place by far I've seen this.